securedrop/sass/source.sass

```diff
@@ -180,8 +180,8 @@ p#codename
   .confirm-prompt
     position: absolute
     width: 300px
-    right: 5%
-    top: -9%
+    right: 10%
+    top: -20%
```

Comment:
When I look at current `develop` I agree that the delete reply confirmation does not render well in Tor Browser:
 
![develop](https://user-images.githubusercontent.com/7832803/27655010-b2b9b046-5bf8-11e7-9f6d-e552d6c83f17.png)

With your PR, I see: 
![your-pr](https://user-images.githubusercontent.com/7832803/27655039-d794cd1a-5bf8-11e7-8d98-2428b5773d3b.png)

which definitely looks better :+1:. 

However if we set `top: 0%` (can just remove the line since it's the default behavior) then it's also nicely centered vertically:

![my modification](https://user-images.githubusercontent.com/7832803/27655057-e48534e2-5bf8-11e7-9e79-a3a48a788114.png)

Let me know what you think!